tests/penndata/test_views.py

```diff
@@ -109,13 +107,13 @@ def test_first_response(self):
         self.assertEqual(res_json[3]["type"], "laundry")
```

Comment:
I know it's not part of the PR, but a few lines above this you can use `self.assertIn` to simplify some of the tests